<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Z_ModeToolWid</class>
 <widget class="QWidget" name="Z_ModeToolWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="serialBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>93</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>串口设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="serialLab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>26</y>
     <width>133</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>COM0  未打开</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="settingBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>设备设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>启动测试</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>SNMP测试</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RTU 测试</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>测试模式</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.1.163</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
